### ISSUE TITLE
lr_mul, wd_mul params probelms in optimizer.py

### ISSUE THREAD

**Issue by xzqjack** [dmlc/mxnet#2921]
Hi, functions `set_lr_mult` and `set_wd_mult` in `optimizer.py` are designed to set multi learning rates and multi wd rates. However, i haven't found any place call these functions except in function `__init__` with input `{}`. In other words, i don't know how to set multi learning rate with current code. 
So I change the `__init__` function to invoke  functions `set_lr_mult` and `set_wd_mult` and it works. Here is the changed code:
```
    def __init__(self, rescale_grad=1., param_idx2name=None, wd=0.,
                 clip_gradient=None, learning_rate=0.01, lr_mult=None, wd_mult=None,
                 lr_scheduler=None, sym=None, begin_num_update=0):
        self.lr = learning_rate   
        self.wd = wd
        if lr_mult is None: lr_mult = {}
        if wd_mult is None: wd_mult = {}
        self.set_lr_mult(lr_mult)
        self.set_wd_mult(wd_mult)
```

I don't know whether it is bug. 
If no, please show me some hits or example code to set multi_rate. 
Thank you!

**Comment by sodabeta7** [comment]
You should specify ``lr_mult`` in corresponding symbol's attribute.
``set_lr_mult`` and ``set_wd_mult`` are invoked in ``__init__`` to construct a dictionary named ``self.lr_mult`` which is used to determine specified learning rate of params.

**Comment by xzqjack** [comment]
@sodabeta7 
Thank  you. I find the documents about `attr` of  symbols' attribute.
